libraries/BotKit Samples/Microsoft.Bot.Sample.Slack/SimpleSlackAdapterOptions.cs

```diff
@@ -29,12 +29,12 @@ public SimpleSlackAdapterOptions(string verificationToken, string botToken)
 
         public string RedirectUri { get; set; }
```

Comment:
Shouldn't this property have a _summary/comment_? Or these kind of changes go in another PR?

Reply:
I think StyleCop doesn't require these because they are already documented in the interface which they belong.